src/state/routing/slice.ts

```diff
@@ -146,8 +138,7 @@ export const routingApi = createApi({
               tokenOut: tokenOutAddress,
               amount,
               type,
-              // if forceUniswapXOn is not ON, then use the backend's default value
-              useUniswapX: forceUniswapXOn || undefined,
+              useUniswapX: true,
```

Comment:
I think this should be undefined here based on the previous comment? `if forceUniswapXOn is not ON, then use the backend's default value` tagging @tinaszheng

Reply:
yeah i agree, we should just remove this